CMakeLists.txt

```diff
@@ -24,61 +27,20 @@ endif ()
 
 find_package(MPI REQUIRED)
 
-# Add include paths
-if (FIBER_ENABLE_HEFFTE)
-add_definitions(-DFIBER_ENABLE_HEFFTE)
-include_directories($ENV{HOME}/heffte/build/include)
-link_directories($ENV{HOME}/heffte/build)
-endif()
-
-if (FIBER_ENABLE_FFTMPI)
-add_definitions(-DFIBER_ENABLE_FFTMPI)
-include_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-link_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-endif()
-
-if (FIBER_ENABLE_ACCFFT)
-add_definitions(-DFIBER_ENABLE_ACCFFT)
-include_directories($ENV{HOME}/Benchmarks_FFT/accfft/build/include)
-link_directories($ENV{HOME}/Benchmarks_FFT/accfft/build)
-endif()
-
-if (FIBER_ENABLE_P3DFFT)
-add_definitions(-DFIBER_ENABLE_P3DFFT)
-include_directories($ENV{HOME}/Benchmarks_FFT/p3dfft/include)
-link_directories($ENV{HOME}/Benchmarks_FFT/p3dfft/build)
-endif()
+# Location of the FFT library
+include_directories(${FIBER_FFT_INCLUDE_DIR})
+link_directories(${FIBER_FFT_LIB_DIR})
```

Comment:
To link multiple libraries, sometimes multiple directories are required like this: `-DFIBER_FFT_LIB_DIR="/path/to/bheffte;/path/to/ffte-7.0"` I would suggest to rename it to to `_DIRS` or `_DIRECTORIES` suffix. Both are used across the CMake universe but the latter seems to be more closer to the canon.

Reply:
Yes, we would be able to specify multiple directories this way.  I agree that using "...DIRS" would make this more clear.  Obviously, we would also prefer to be able to automatically search of the libraries, but I this this is preferable to using hard-coded paths or to using separate flags for every library.